### DISCUSSION TITLE
Making the pause button on bluetooth headphones work with firefox?

### DISCUSSION THREAD

**Discussion by X64321** [i3/i3#5614] [Q&A]
I'm running the Cinnamon edition of Mint and am currently stripping  it down and switching it to i3. One of the things that has been giving me problems is my Bluetooth headphones being able to pause YouTube videos. The pause works just fine in Cinnamon but doesn't seem to work at all in i3. Does anyone have insight into how I could fix this?

**Comment by orestisfl** [comment]
The bluetooth headphone buttons probably produce normal x11 keybind events that you need to map. Try

```
bindsym XF86AudioPlay exec --no-startup-id playerctl play-pause
bindsym XF86AudioPause exec --no-startup-id playerctl play-pause
bindsym XF86AudioNext exec --no-startup-id playerctl next
bindsym XF86AudioPrev exec --no-startup-id playerctl previous
```

and install [playerctl](https://github.com/altdesktop/playerctl)

See also: https://github.com/i3/i3/discussions/5618
